src/rudderstack.ts

```diff
@@ -56,6 +62,24 @@ export class RudderStack {
             this.analytics.ready(() => {
                 this.has_initialized = true
                 this.has_identified = !!(this.getUserId() || this.getAnonymousId())
+                let eventQueue: AnalyticsEvent[] = []
```

Comment:
Would you please put this to a function to make the ready function more readable?